.github/workflows/ci.yml

```diff
@@ -151,6 +151,18 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
+      postgres13:
+        image: postgres:13.1
+        env:
+          POSTGRES_DB: test
+          POSTGRES_PASSWORD: pass
+        ports:
+          - 5434:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

Comment:
Please add this to the migration workflow (line 260~) as well (GitHub doesn't support YAML anchors).